corva/network/api.py

```diff
@@ -6,23 +6,23 @@
 from requests.adapters import HTTPAdapter
 from urllib3 import Retry
 
-from corva import settings
-
 
 class Api:
     ALLOWED_METHODS = {'GET', 'POST', 'PATCH', 'PUT', 'DELETE'}
+    DEFAULT_TIMEOUT = 600
```

Comment:
Shouldn't these go into settings?

Reply:
Settings contain only env variables as described [here](https://www.notion.so/corva/Data-Apps-454ddb3fdb174e10b5cd53679ad999ca).
`DEFAULT_TIMEOUT` belongs to `Api` and is not meant to be changed.
User can set custom default timeout on `Api` initialization like `Api(timeout=601)`